### PR TITLE
Unit 1  Lesson 1  Ruby

### PR DESCRIPTION
**Pull request by aviflax**
Pretty happy with how this turned out.

To run, fetch and check out the branch, navigate to the dir, and run `./start.sh`.

### CODE REVIEW

File: unit_1/lesson_1/ruby/lesson.rb [hunk 1]
```diff
@@ -14,23 +14,25 @@ def self.prompt(str)
...
-    paths = Dir.glob 'fragment_*.rb'
+    paths = Dir.glob('fragment_*.rb').sort
     paths[0..-2].map { |path| [path, false] }
                 .push([paths[-1], true])
   end
 
   def self.start
     print "\n"
-    fragments.each { |path, is_last| do_fragment IO.read(path), is_last }
+    bind = binding
```
Comment (arsenerei) [inline unit_1/lesson_1/ruby/lesson.rb]: What is `binding`?

Comment (aviflax) [reply unit_1/lesson_1/ruby/lesson.rb]: See [`Kernel#binding`](https://ruby-doc.org/core-2.5.0/Kernel.html#method-i-binding) and [`Binding`](https://ruby-doc.org/core-2.5.0/Binding.html).

**Comment by aviflax** [comment]
Thanks for the review!

File: unit_1/lesson_1/ruby/lesson.rb (hunk 1 above)
Comment (arsenerei) [reply unit_1/lesson_1/ruby/lesson.rb]: Oh, nice!

**Pull request merged**